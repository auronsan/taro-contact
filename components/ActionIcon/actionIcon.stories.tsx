import React from 'react';
import { ActionIcon } from './index';

export default {
  title: 'components/ActionIcon',
  component: ActionIcon,
};

export const withOnClick = () => <ActionIcon onClick={() => {}}>Click Me</ActionIcon>;
