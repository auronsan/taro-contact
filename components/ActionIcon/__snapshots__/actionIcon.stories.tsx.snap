// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/ActionIcon withOnClick smoke-test 1`] = `
<main class="roboto-normal">
  <div style="display: flex; justify-content: center; align-items: center; height: 100vh; flex-direction: column;">
    <div class="actionIcon_root__jjWKK"
         role="button"
    >
      Click Me
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
